Sources/PathControl/StandardPathControl.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  StandardPathControl.swift
+//  PathControl
+//
+//  Created by Vincent LIEGEOIS on 25/10/2024.
+//
+
+import SwiftUI
+
+/// A control for display of a file system path or virtual path information.
+public struct StandardPathControl: NSViewRepresentable {
+
+    @Binding private var url: URL?
+
+    /// Creates a standard path control.
+    ///
+    /// - Parameter url: A binding to property that defines the currently-selected url.
+    public init(url: Binding<URL?>) {
```

Comment:
From the looks of it `standard` `NSPathControl` with `nil` url doesn't display anything. So it accepting optional doesn't make sense to me here. I'd prefer to have this and `StandardStaticPathControl` operate on non-optional url. Unless I'm missing use case?

```suggestion
    public init(url: Binding<URL>) {
```